web/themes/custom/camp/templates/block/block--footer_block.html.twig

```diff
@@ -0,0 +1,87 @@
+ *   - id: A valid HTML ID and guaranteed unique.
+ * - title_attributes: Same as attributes, except applied to the main title
+ *   tag that appears in the template.
+ * - title_prefix: Additional output populated by modules, intended to be
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ *
+ * @see template_preprocess_block()
+ *
+ * @ingroup themeable
+ */
+#}
+<div class="footer">
+  <div class="row footer-follow">
+    <div class="span-2 footer-col-1">
+      <i class="icon-logo_alt icon-logo_alt--footer "></i>
+      {{ content.footer_column_1 }}
+    </div>
+    <div class="span-5 follow-cell footer-col-2">
+      {{ content.footer_column_2 }}
+      <div class="form-email-forside">
+        <!-- Begin MailChimp Signup Form -->
+        <div id="mc_embed_signup" class="mc_embed_signup">
+            <form action="//sl2017.us12.list-manage.com/subscribe/post?u=ff41604c0441f9e27f87a47e2&amp;id=c1834121e4" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
+                <div id="mc_embed_signup_scroll">
+
+                    <div class="mc-field-group">
+                        <input type="email" value="" name="EMAIL" class="required email mc-input" id="mce-EMAIL" placeholder="Email Adresse">
+                    </div>
+                    <div class="mc-field-group">
+                        <input type="text" value="" name="FNAME" class="required mc-input" id="mce-FNAME" placeholder="Fornavn">
+                    </div>
+                    <div id="mce-responses" class="clear">
+                        <div class="response" id="mce-error-response" style="display:none"></div>
+                        <div class="response" id="mce-success-response" style="display:none"></div>
+                    </div>    <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
+                    <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_ff41604c0441f9e27f87a47e2_c1834121e4" tabindex="-1" value=""></div>
+                    <div class="mc-field-group"><input type="submit" value="Tilmeld" name="subscribe" id="mc-embedded-subscribe" class="button mc-submit"></div>
+                </div>
+            </form>
+        </div>
+        <script type='text/javascript' src='//s3.amazonaws.com/downloads.mailchimp.com/js/mc-validate.js'></script><script type='text/javascript'>(function($) {window.fnames = new Array(); window.ftypes = new Array();fnames[0]='EMAIL';ftypes[0]='email';fnames[1]='FNAME';ftypes[1]='text';fnames[2]='LNAME';ftypes[2]='text'; /*
+         * Translated default messages for the $ validation plugin.
+         * Locale: DA
+         */
+            $.extend($.validator.messages, {
```

Comment:
og detaljer her: http://kb.mailchimp.com/lists/signup-forms/translate-the-mailchimp-embed-code